install.sh

```diff
@@ -117,7 +118,7 @@ do_setup() {
   setup_tools
   setup_vim $HVN_DEST
 
-  if test -z "$BASIC_ONLY"
+  if "$FULL_INSTALL"
```

Comment:
Doing a string comparison here instead of `true`/`false` commands would be clearer.  See this link: http://stackoverflow.com/a/2953673